Makefile

```diff
@@ -10,6 +10,24 @@ unit_test:
 fix:
 	pipenv run pytest -x tests
 
+
+# List of all integration test scenarios
+integration_test_scenarios := $(wildcard tests/integration/*/runme.sh)
+
+# Default opera executable (tailored for developers). Can be overridden by
+# setting the OPERA environment variable to something else.
+OPERA ?= "pipenv run opera"
+
+# Point pipenv to the right file since we can execute tests from a subdir.
+export PIPENV_PIPFILE := $(realpath Pipfile)
+
```

Comment:
I would suggest that we move those line to the top of the Makefile in order to keep the configuration and target definitions separated a but more.

Reply:
Yeah sure, this would be nicer.